intel-sgx/ppid-retrieval-tool/Docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Stage 0: A Intel SGX SDK container
+FROM ubuntu:24.04 AS sgx_sdk
+
+## Install user
+RUN useradd -rm -d /home/ppid-tool -s /bin/bash -g root -G sudo -u 1010 ppid-tool
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN echo 'ppid-tool:ppid-tool' | chpasswd
+USER ppid-tool
+WORKDIR /home/ppid-tool
+
+## Install SGX Dev tools
+USER root
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    gnupg \
+    wget \
+    sudo
+
+## App build time dependencies
+RUN apt-get update
+RUN apt-get install -y build-essential
+
+WORKDIR /opt/intel
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.25/distro/ubuntu24.04-server/sgx_linux_x64_sdk_2.25.100.3.bin
+RUN chmod +x sgx_linux_x64_sdk_2.25.100.3.bin
+RUN echo 'yes' | ./sgx_linux_x64_sdk_2.25.100.3.bin
+
+## Install SGX runtime libraries
+USER root
+RUN echo 'deb [signed-by=/etc/apt/keyrings/intel-sgx-keyring.asc arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu noble main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+RUN wget https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key
+RUN echo 92f96f84281031d889deb81060c44325f0481aee621ae47a15ae1df4431b4a23 intel-sgx-deb.key | sha256sum -c
+RUN cat intel-sgx-deb.key | sudo tee /etc/apt/keyrings/intel-sgx-keyring.asc > /dev/null
+RUN apt-get update
+RUN apt-get install -y libsgx-urts
```

Comment:
Another approach is using the pre-built dynamic libraries in https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntu24.04-server/PCKIDRetrievalTool_v1.22.100.3.tar.gz

Reply:
```
~/Downloads/PCKIDRetrievalTool_v1.22.100.3 
> ls
libmpa_uefi.so.1	       libsgx_pce.signed.so.1  network_setting.conf
libsgx_enclave_common.so.1     libsgx_urts.so	       PCKIDRetrievalTool
libsgx_id_enclave.signed.so.1  License.txt	       README.txt
```

Reply:
For build stage , current approach is fine.
But for run stage below, I suggest to use this